### ISSUE TITLE
Unable to log in using Apple ID credentials

### ISSUE THREAD

**Issue by uajqq** [shahruz/unofficialrss#1]
I log in to my Stitcher Premium account using my Apple ID, but this option is not available when I select "Log in to Stitcher to generate your own RSS feeds" on UnofficialRSS v2.0. I can log in through the Stitcher website normally, but even if I am logged into Stitcher, I am presented with a login prompt that lacks Apple ID support when I try to link UnofficialRSS.

**Comment by j0be** [comment]
Yeah, this seems to be a limitation of the Stitcher OAuth service. There's nothing Unofficial Rss can do to circumvent that.

**Comment by joshg253** [comment]
Seeing similar using Google account login.
